modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -262,6 +262,35 @@ describe('User CRUD tests', function () {
     });
   });
 
+  it('should be able to change profile picture if signed in', function (done) {
```

Comment:
Can we also add a test for not being able to change profile picture if not signed in.